website/src/models/group.js

```diff
@@ -84,6 +84,31 @@ GroupSchema.pre('save', function(next){
   next();
 })
 
+GroupSchema.pre('remove', function(next) {
```

Comment:
the only thing @TheHollidayInn is that this maybe should be in a post middleware rather than a pre... I'll fix it in develop